src/client/java/teammates/client/scripts/scalabilitytests/InstructorCourseEnrollPageScaleTest.java

```diff
@@ -0,0 +1,94 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.testng.annotations.Test;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.Logger;
+import teammates.client.scripts.util.Stopwatch;
+import teammates.test.cases.browsertests.BaseUiTestCase;
+import teammates.test.pageobjects.InstructorCourseEnrollPage;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Covers 'enroll' view for instructors. Run 'instructorCourseEnrollPageScaleTestDataGenerator.py' before running the tests.
+ * SUT: {@link InstructorCourseEnrollPage}.
```

Comment:
Let's organize the doc this way:
```java
/**
 * SUT: {@link Const.ActionURIs#INSTRUCTOR_COURSE_ENROLL_PAGE}.
 *
 * <p>Run InstructorCourseEnrollPageScaleTestDataGenerator.java before running the tests. 
 */
```
- The Page Object is not the SUT; the page in the address is.
- We're using Java for everything, right?